script.js

```diff
@@ -0,0 +1,24 @@
+const currencyRate = {
+  timestamp: 1620386223,
+  base: "DKK",
+  rates: {
+    USD: 0.1444,
+    EUR: 0.134,
+    GBP: 0.1151,
+    CAD: 0.1963,
+    JPY: 20.9142,
+    DKK: 1,
+  },
+};
+
+function onSubmit() {
+  const amountIn = document.getElementById("amount_input").valueAsNumber;
+  const currencyIn = document.getElementById("curr_input").value;
```

Comment:
You are not using this value; instead, the rate is still calculated based on DKK. Is this intentional?

Reply:
Since its not used you can delete `const currencyIn = document.getElementById("curr_input").value;`